cp2/ControlPanel/ControlPanel/models.py

```diff
@@ -19,42 +19,39 @@ def user_created_handler(sender, instance, *args, **kwargs):
     """
     Signal pre-save in djagno DB will try to update the editor with given email if it exists.
     """
+    new_editor = instance
```

Comment:
Only the sender is passed as a positional argument to signal receivers. Since it is a keyword argument I haven't found a way to change it directly in the function argument.